ACT-Assignment/ViewControllers/LoginViewController.swift

```diff
@@ -9,11 +9,88 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet var backgroundView: UIView!
+    @IBOutlet var emailView: UIView!
+    @IBOutlet var passwordView: UIView!
+    @IBOutlet var eyeButton: UIButton!
+    @IBOutlet var emailTextField: UITextField!
+    @IBOutlet var passwordTextField: UITextField!
+    @IBOutlet var loginButton: UIButton!
+    
+    var eyeFlipper = 0
```

Comment:
All the IBOutlets must be private. Other properties also should have access specifier.